docs/concepts/copilot/copilot.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Data Copilot (NEW!)
+title: Data Copilot
```

Comment:
fyi I removed "new" per Scott's request. Maybe related to SEO, I didn't catch the reason.